samples/bundle-reactor-deploy/README.md

```diff
@@ -10,7 +10,7 @@ This sample is the quickest way to give the Maven plugin a try without having an
 ## Set Up
 ### Have your system programmer create your bundle definition in CSD
 Your system programmer should create a bundle definition in CSD and tell you the CSD group and bundle definition name they have used.
-The bundle directory of your bundle definition should be set as follows: `<bundle_deploy_root>/<bundle_id>_<bundle_version>`.  So for this sample, if your bundle_deploy_root was `/u/someuser/bundles/`, the bundle directory would be `/u/someuser/bundles/demo-bundle_0.0.1`.
+The BUNDLEDIR of the bundle definition your system programmer creates should be set as follows: `<bundles-directory>/<bundle_id>_<bundle_version>`.  So for this sample, if your system programmer configured `bundles-directory` as `/u/someuser/bundles/`, the BUNDLEDIR would be `/u/someuser/bundles/demo-bundle_0.0.1`.
```

Comment:
We use "BUNDLE definition" in KC, e.g. https://www-03preprod.ibm.com/support/knowledgecenter/en/SSGMCP_5.5.0/configuring/cpsm/eyuac_viewingbundles.html
Shall we keep consistent?

Reply:
OK - I don't really like the capital letters and worry about all the other places we might not have used them but have made this change!